CHANGELOG.md

```diff
@@ -1,75 +1,83 @@
+The default labels are now:
 
-This was added in [pull request #1925: Generate fixtures.json files for components on build:package](https://github.com/alphagov/govuk-frontend/pull/1925)
+- **Navigation menu** for `navigationLabel`
+- **Show or hide navigation menu** for `menuButtonLabel`
 
-#### Add new brand colour for The Foreign, Commonwealth and Development Office (FCDO)
+This was added in pull requests:
 
-This was added in [pull request #1918: Add new brand colour for FCDO](https://github.com/alphagov/govuk-frontend/pull/1918).
+- [#1905: Set navigation and mobile menu labels of the header component with new options](https://github.com/alphagov/govuk-frontend/pull/1905)
+- [#1943: Change header menu button label](https://github.com/alphagov/govuk-frontend/pull/1943)
 
-#### Render header navigation item without a link
+Thanks to [@domoscargin](https://github.com/domoscargin) for raising this issue.
 
-You can now render navigation items using the header macro without wrapping them in a link.
+##### Add navigation items without links
 
-You can do this by setting `text` or `html` and not setting `href`.
+To add a navigation item without a link, use the `text` or `html` option to add the item but do not use the `href` option.
 
 For example:
 
 ```javascript
 {{ govukHeader({
-  navigation: [
+    navigation: [
     {
-      html: '<form method="post" action="url.com">
-              <input type="submit" class="app-logout-button-style" value="Log out" />
-            </form>'
+      html: "<form method='post' action='url.com'>
+              <input type='submit' class='app-logout-button-style' value='Log out' />
+            </form>"
     }
   ]
 }) }}
 ```
 
-This was added in [pull request # 1921: Make it possible to exclude link from header navigation item](https://github.com/alphagov/govuk-frontend/pull/1921).
+This was added in [pull request #1921: Make it possible to exclude link from header navigation item](https://github.com/alphagov/govuk-frontend/pull/1921).
 
-#### Set navigation and mobile menu labels of the header with new options
+#### Test if your HTML matches GOV.UK Frontend
 
-You can now customise the `aria-label` attributes of the navigation and mobile menu of the header component using the new `navigationLabel` and `menuButtonLabel` options.
+You can now use our test fixtures to [check you're outputting the same HTML that GOV.UK Frontend uses](http://frontend.design-system.service.gov.uk/testing-your-html/).
 
-For example:
+This was added in [pull request #1925: Generate fixtures.json files for components on build:package](https://github.com/alphagov/govuk-frontend/pull/1925). Thanks to everyone who fed back on [our test fixtures proposal](https://github.com/alphagov/govuk-frontend/issues/1830#issuecomment-665075842).
 
-```javascript
-{{ govukHeader({
-    navigationLabel: "Custom navigation label",
-    menuButtonLabel: "Custom menu label"
-}) }}
-```
+### Fixes
```

Comment:
This section appears to be duplicated.

Reply:
I think (?) I just accidentally fixed this when I rebased... let me know if not though!

Reply:
Ah wait no sorry, it's still there... I'll fix

Reply:
Should be fixed now - thanks for spotting!